packages/atlas-anatomogram-experiment-table/src/anatomogramExperimentTable.js

```diff
@@ -0,0 +1,313 @@
+      } else if (view === `cells`) {
+        requestOntologyIds = [
+          `CL_2000002`, `EFO_0010710`, `CL_3000001`, `UBERON_0000371`, `UBERON_0000319`, `UBERON_0000426`,
+          `EFO_0010712`, `EFO_0010711`, `CL_0002601`, `EFO_0010708`
+        ]
+      }
+    }
+
+    this.setState({
+      resource:
+        URI(`json/experiments/${this.props.experimentAccession}/marker-genes/cell-types`, this.props.host)
+          .search({ organismPart: requestOntologyIds.map(ontologyAccessionToOntologyUri).join(`,`) })
+          .toString()
+    })
+  }
+
+  handleResponse(json) {
+    this.setState(() => ({
+      experiments: json
+    }));
+    this.forceUpdate();
+  }
+
+  _showLinkBoxIds(id) {
+    this.setState({
+      showIds: [...new Set(id.concat(this.state.showIds))]
+    })
+  }
+
+  _clearSelectedIds() {
+    this.setState({
+      showIds: this.props.showIds,
+      selectIds: [],
+      highlightIds: []
+    })
+  }
+
+  render() {
+    const { anatomogramHost, dataHost, organs, description } = this.props
+    return (
+      <div>
+        <div className={`row`}>
+          <div className={`large-3 small-12 medium-3 columns`}>
+            {
+              organs.length > 1 &&
+              <div className={`small-6 small-centered columns`}>
+                <select value={this.state.selectedSpecies} onChange={this._handleSelectOnChange}>
+                  {organs.map((organ) => {
+                    return <option key={organ} value={organ}>{capitalizeFirstLetter(organ.replace(`_`, ` `))}</option>
+                  })}
+                </select>
+              </div>
+            }
+
+            <Anatomogram
+              species={this.state.selectedSpecies}
+              showIds={this.state.showIds}
+              highlightIds={this.state.highlightIds}
+              selectIds={this.state.selectIds}
+              selectAllIds={this.state.selectAllIds}
+              clearSelectedIds={this._clearSelectedIds}
+              onClick={this._addRemoveFromSelectIds}
+              showLinkBoxIds={this._showLinkBoxIds}
+              afterSwitchView={this._afterSwitchView}
+              organs={organs}
+              atlasUrl={anatomogramHost}
+            />
+          </div>
+          <div className={`large-9 small-12 medium-3 columns`}>
+            <div>
+              <p>{description}</p>
+            </div>
+          </div>
+
+          <div>
+            <TableManager
+              host={dataHost}
+              resource={this.state.resource}
+              dataRows={this.state.experiments}
+
+              tableHeaders={[
+                {
+                  label: `Load date`,
+                  dataKey: `loadDate`,
+                  sortable: true,
+                  width: 0.5
+                },
+                {
+                  label: `Species`,
+                  dataKey: `species`,
+                  searchable: true,
+                  sortable: true
+                },

```

Comment:
Since we’re going with only human experiments I don’t know if we need this one. Definitely not `searchable` or `sortable`. Do we have plans of including non-human experiments?